---
author: "Daniel Antonio Negrón, PhD"
geometry: vmargin=0.75in,hmargin=0.5in
header-includes:
- \usepackage{fancyhdr}
- \usepackage{framed}
- \usepackage{pmboxdraw}
- \pagestyle{fancy}
- \fancyhead[L]{\large \textbf{Daniel Antonio Negrón, PhD}}
- \fancyhead[C]{Scientist}
- \fancyhead[R]{dnegron2@gmu.edu}
- \fancyfoot[R]{\thepage}
- \setlength{\headheight}{24pt}
- \headsep=24pt
- \fancyfoot[C]{https://dnanto.github.io/}
output: pdf_document
---

```{r setup, include=F}
library(dplyr)
library(stringr)
```

`> Multidisciplinary scientist and polyglot programmer solving problems in academia and industry. _`

## Skills \dotfill

\begin{oframed}
\textbf{Biology} \hfill alignment, annotation, homology modeling, microarrays, omics, sequencing, PCR/LAMP, phylogenetics \break
\textbf{Database} \hfill BLAST+, Cypher, ER modeling, Gremlin, MySQL, PostgreSQL, SQL, SQLite, SPARQL, triplestore \break
\textbf{Computing} \hfill cloud, compression, data structures, GPU, HPC, microcontrollers, numerical, parallel, threading, UI \break
\textbf{Reproducibility} \hfill Conda, Docker, Git, \LaTeX, pip, poetry, Podman, R Markdown, Snakemake, Spack, uv, Zotero \break
\textbf{Programming} \hfill AWK, BASH, C, C++, C\#, CUDA, GLSL, HTML, Java, JavaScript, jq, perl, node, python \break
\textbf{Mathematics} \hfill calculus, geometry, linear algebra, MCMC, ML, signed-distance functions, statistics \break
\textbf{Language} \hfill Spanish (co-native), Japanese (JLPT 5)
\end{oframed}

## Education \dotfill

\begin{oframed}
\textbf{George Mason University, School of Systems Biology} \hfill Fairfax, VA
\begin{leftbar}
\textit{PhD in Bioinformatics \& Computational Biology} \hfill Aug 2014 – May 2021 \break
Dissertation: Molecular Clock Analysis of Human Adenovirus \hfill \break
Chair: Dr. Donald Seto; Co-Chairs: Dr. Patrick Gillevet \& Dr. Sterling Thomas \hfill
\end{leftbar}

\textbf{Johns Hopkins University, Zanvyl Krieger School of Arts \& Sciences} \hfill Baltimore, MD
\begin{leftbar}
\textit{MS in Bioinformatics} \hfill Nov 2009 – Dec 2011 \break
Capstone: Parallel Algorithm for \textit{de novo} Co-Expression Networks from Microarray Data \hfill \break
Adviser: Dr. Thomas Koval; Supervisors: Dr. Sterling Thomas \& Dr. Catherine E. Campbell \hfill
\end{leftbar}

\textbf{Boston University, College of Arts \& Sciences} \hfill Boston, MA
\begin{leftbar}
\textit{BA in Biochemistry \& Molecular Biology, Minor in Computer Science} \hfill Sep 2005 – May 2009 \break
Project: Microsatellite junction counts are inversely proportional to unit size, gap, and edit distance \hfill \break
Adviser: Dr. Gary Benson \hfill \break
Project: Type-II superconductor simulator for magnetic vortex formation \hfill \break
Adviser: Dr. George Ogurek Zimmerman \hfill
\end{leftbar}
\end{oframed}

## Experience \dotfill

\begin{oframed}
\textbf{Noblis, Inc.} \hfill Reston, VA
\begin{leftbar}
\textit{Scientist} \hfill Oct 2011 – Present \break
Develop algorithms and program applications for the design and evaluation of molecular diagnostics such as PCR/LAMP assays and multiplexed antigen tests for biosurveillance with refinements based on BSL-II experimentation, primarily supporting DTRA. Analyze laboratory data for microbial based food and fuel production. Consult on interdisciplinary client work across corporate mission areas, internal R\&D, and grant/proposal/patent submissions. Integrate advanced algorithms, machine learning, and statistics into reproducible and portable scientific workflows to generate publication quality reports. Design and supervise internship projects. \hfill \break
Recent and ongoing research projects include a phone app to interpret colorimetric LAMP assay and lateral flow immunoassay results using OpenCV, 3D-printed robot arm with computer vision for wet lab task automation (as PI), DNA-storage algorithm development (with patent submitted and experiment sent to the International Space Station via NASA's SpaceX 33rd Commercial Resupply Service), and AI-directed protein design with RFDiffusion, ProteinMPNN, AlphaFold, and laboratory validation. \hfill \break
As a cross-disciplinary scientist, past projects included microarray analysis, full-stack laboratory information management system (LIMS) development, expert system for well water remediation, stock trade automation with graph-based correlation strategy, traffic congestion prediction, dark web social network analysis (supporting INTERPOL), statistical fingerprinting of nuclear fuel rods, forensic reverse-engineering of file formats, containerized computer virus scanning microservice frameworks (as PI), and efficient language detection on low-energy hardware. \hfill
\end{leftbar}
\begin{leftbar}
\textit{On-Call Technical Intern} \hfill Aug 2011 – Oct 2011 \break
Port algorithms to run on the CRAY XMT 2 supercomputer using the C programming language and compiler directives. Analyze \textit{P. aeruginosa} microarray data to identify virulence factors. Also analyzed protein microarray data of hemorrhagic fever viruses from the BSL-III laboratory data from GMU. \hfill
\end{leftbar}
\begin{leftbar}
\textit{Noblis/JHU Independent Research Project Fellowship} \hfill Jun 2011 – Aug 2011 \break
Upgrade an algorithm for the analysis of host-pathogen interaction networks from microarray data. Develop an MPI-enabled version for parallel execution. Analyze human-plague (\textit{Y. pestis}) microarray data to identify network motifs. \hfill
\end{leftbar}

\textbf{Infant \& Toddler Connection of Fairfax-Falls Church} \hfill Fairfax, VA
\begin{leftbar}
\textit{Administrative Assistant II / Data Analyst} \hfill Sep 2009 – Jun 2011 \break
Develop VBA macros for use with Microsoft Office applications. Automate administrative tasks using Windows batch scripts. Manage IT support at the Falls Church satellite site. \hfill
\end{leftbar}

\textbf{Boston University} \hfill Boston, MA
\begin{leftbar}
\textit{Laboratory for Biocomputing and Informatics} \hfill Sep 2008 – May 2009 \break
Conduct research under the supervision of Dr. Gary Benson. Implement algorithms in Java and python using distance scoring, sequence alignment, and spaced-seed searching to analyze microsatellite junctions and tandem repeat evolution. \hfill \break
\textit{Experimental Condensed Matter Physics} \hfill Sep 2008 – May 2009 \break
Conduct research under the supervision of Dr. George Ogurek Zimmerman. Program a type-II superconductor simulator to visualize phenomena related to magnetic field vortices in Java and python. Collaborate on the 2D implementation of the simulator and subsequently developed the 3D version. \hfill
\end{leftbar}
\end{oframed}

## Licenses & Certifications

\begin{oframed}
\textbf{Nvidia Corporation} \hfill Online
\begin{leftbar}
\textit{Fundamentals of Accelerated Computing with Modern CUDA C++} \hfill Aug 2025 \break
For demonstrating competence in the completion of "Fundamentals of Accelerated Computing with Modern CUDA C++" (Certification ID: dnp-qkQNTBCykgtHv703Tw). \hfill
\end{leftbar}
\end{oframed}

## Awards \dotfill

\begin{oframed}
\textbf{Noblis, Inc.} \hfill Reston, VA
\begin{leftbar}
\textit{Noblis Award for Excellence in Science and Technology} \hfill 2023 \break
For outstanding thought leadership and scientific contributions that advance and solve complex challenges on missions ranging from biodefense to cyber defense. \hfill
\end{leftbar}

\begin{leftbar}
\textit{Board of Trustees' Award for Excellence} \hfill 2021 \break
For contributions to the Northern Virginia Innovation Exchange (NVIX). \hfill
\end{leftbar}

\begin{leftbar}
\textit{Noblis Hackathon Honorable Mention} \hfill 2017  \break
Participated as part of team "Ghost in the Shell Script" in the 24-hr hackathon to improve the search algorithm of the RASMAS medical recall system. The Java solution employed an n-gram indexing strategy for calculating Jaccard similarity. \hfill
\end{leftbar}
\end{oframed}

## Projects \dotfill

\begin{oframed}
\textbf{Homebrewing} \hfill
\begin{leftbar}
\textit{Makgeolli} \hfill Jan 2023 - Present \break
Brew traditional Korean rice wine. Developed an Arduino fermentation monitoring system combining an optoisolator and digital scale to track airlock bubble evolution and mass change over time and write logs to an SD card. Received funds from the Noblis Sponsored Research program to sequence samples and develop LAMP assays for microbes critical to the multiple parallel fermentation process. Results were presented as a talk at ASM 2024 in Atlanta, GA. \hfill
\end{leftbar}

\textbf{Capsid Modeling} \hfill
\begin{leftbar}
\textit{Render viral capsids using Caspar-Klug Theory} \hfill May 2020 – Present \break
In the browser with JavaScript to export SVG diagrams (democapsid @ https://dnanto.github.io/democapsid/app.html). On the GPU with GLSL for realtime renders (protocapsid @ https://www.shadertoy.com/view/dlsGRH). \hfill
\end{leftbar}

\textbf{District Broomball Statistics} \hfill
\begin{leftbar}
\textit{Database and website for league statistics} \hfill Jun 2019 – Present \break
Design ER model, SQLite database, and R code with novel assist network visualizations for league statistics and website generation (https://github.com/dnanto/broomball). \hfill
\end{leftbar}
\end{oframed}

## Publications \dotfill

```{r publications, results='asis', echo=F}
c("paper", "preprint", "patent", "poster", "presentation") %>%
  lapply(function(ele) {
    c(
      paste0("### ", str_to_title(ele), "s"),
      "\\begin{oframed}",
      RefManageR::ReadBib(file.path("data", "cv", paste0(ele, ".bib"))) %>%
        print(.opts = list(style = "markdown", sorting = "ydnt", max.names = Inf)) %>%
        capture.output(),
      "\\end{oframed}",
      "\n"
    )
  }) %>%
  unlist() %>%
  str_flatten(collapse = "\n") %>%
  str_replace_all("D. (A. )?Negrón", "\\\\textbf{D. A. Negrón}") %>%
  str_replace_all("_([^_]*)_", "\\\\textit{\\1}") %>%
  str_remove_all("\\s*\\(Visited\\son\\s[0-9]{2}/[0-9]{2}/[0-9]{4}\\)\\.") %>%
  str_replace_all("\\[(.*)\\]\\((.*)\\)", "\\\\href{\\2}{\\1}") %>%
  cat()
```
